src/recovery/mod.rs

```diff
@@ -181,6 +183,8 @@ impl Recovery {
 
             congestion_recovery_start_time: None,
 
+            max_datagram_size: config.max_datagram_size,
```

Comment:
The initial and minimum congestion window values should also change according to the configured value I think.

That is, instead of being constants, they should change dynamically. i.e. `INITIAL_WINDOW` should be replaced by `INITIAL_WINDOW_PACKETS * config.max_datagram_size` and `MINIMUM_WINDOW` should instead be `MINIMUM_WINDOW_PACKETS * r.max_datagram_size` whenever they are used.

Reply:
Thanks. Now INTIAL_WINDOW and MINIMUM_WINDOW is gone and calculated when Recovery is initialized based on the Config max_datagram_size value.